src/ipa_pc/mod.rs

```diff
@@ -94,6 +93,7 @@ impl<G: AffineCurve, D: Digest> InnerProductArgPC<G, D> {
         let check_time = start_timer!(|| "Succinct checking");
 
         let d = vk.supported_degree();
+        let n = vk.comm_key.len();
```

Comment:
n = d + 1. If we have to keep with the supported_degree() of a PCVerifierKey then there is no reason not to explicitly address the length of the verifier key.

Reply:
Now this variable is named key_len and used also in places where we don't have polynomial and its degree. From the code design perspective it is better to have same value computed in same way, isn't it ? This is why I am using key length instead of d + 1, because key length we can use both: on prover and verifier part